src/core/client/admin/components/StoryInfoDrawer/StorySettingsContainer.css

```diff
@@ -0,0 +1,33 @@
+.tabBar {
+  margin-top: var(--spacing-3);
+}
+
+.tab {
+}
+
+.activeTab {
+  border-bottom: 1px solid var(--color-teal-500);
+}
+
+.tabIcon {
+  margin-right: var(--spacing-1);
+}
+
+.configureStory {
+  padding-top: var(--spacing-1);
+  font-weight: var(--font-weight-secondary-regular);
+  color: var(--palette-text-500);
+}
+
+.setting {
+  padding: var(--spacing-3) 0;
+  align-items: center;
+}
+
+.setting > input {
+  vertical-align: bottom;
+}
+
+.checkboxText {
+  font-weight: var(--font-weight-secondary-regular);
```

Comment:
Same as prior comment, make sure to set `font-family-secondary` if using `font-weight-secondary-...` 😄